dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateCopyTestJan2023.tsx

```diff
@@ -0,0 +1,153 @@
+import { cmp } from '@guardian/consent-management-platform';
+import { Button, Link, LinkButton } from '@guardian/source-react-components';
+import { trackLink } from '../componentEventTracking';
+import type { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodySeparator,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+	signInLink,
+} from './shared';
+
+export const SignInGateCopyTestJan2023 = ({
+	signInUrl,
+	guUrl,
+	dismissGate,
+	abTest,
+	ophanComponentId,
+	isMandatory = false,
+}: SignInGateProps) => {
+	const getCopyTextFromVariant: (abVariant: string | undefined) => string = (
+		abVariant,
+	) => {
+		switch (abVariant) {
+			case 'quick-and-easy': {
+				return "Register: it's quick and easy";
+			}
+			case 'take-a-moment': {
+				return 'Take a moment to register';
+			}
+			default: {
+				return 'You need to register to keep reading';
+			}
+		}
+	};
+	return (
+		<div css={signInGateContainer} data-cy="sign-in-gate-main">
+			<style>{hideElementsCss}</style>
+			<div css={firstParagraphOverlay} />
+			<h1 css={headingStyles}>
+				{getCopyTextFromVariant(abTest?.variant)}
+			</h1>
+			<p css={bodyBold}>
+				It’s still free to read - this is not a paywall
```

Comment:
Apostrophe here is fine, but it’s a hyphen-minus and should be an en dash (`–`).

Reply:
Ah, looks like this was an issue in the MainSignInGate which I forked this from, so I've updated both.